raster/src/main/scala/geotrellis/raster/render/ColorMap.scala

```diff
@@ -202,6 +215,8 @@ class IntColorMap(breaksToColors: Map[Int, Int], val options: Options = Options.
 
   private lazy val orderedColors: Vector[Int] = orderedBreaks.map(breaksToColors(_))
   lazy val colors = orderedColors
+  lazy val breaksMap = breaksToColors
+  lazy val breaksMapDouble = Map.empty[Double,Int]
```

Comment:
Why do we need a breaks map? Can't we just implement this functionality as the `toString` method?